### PR TITLE
Update E2ETest

### LINKED ISSUE #1119: E2ETest::test_it_runs_on_itself() is broken
[The build is failing with:](https://travis-ci.org/infection/infection/jobs/658378536#L436)

> Cannot open file ""--exclude-group=integration".php".

This might suggest that [`--test-framework-options` flag](https://infection.github.io/guide/command-line-options.html#test-framework-options) is broken, but it isn't. 

Fixing this above issue we find that it still fails, but with:

> The process has been signaled with signal "11".

This is SIGSEGV, segmentation violation, which is very peculiar.

### PR DESCRIPTION
**Pull request by sanmai**
- Clarify that this test isn't working with PHPDBG anywhere. The same goes e.g. for PHPUnit's `@runInSeparateProcess`.
- Allow running with pcov.
- Do not log progress.
- Fixes #1119
- Closes #1117

### CODE REVIEW

**Comment by sanmai** [comment]
> The job exceeded the maximum time limit for jobs, and has been terminated.

Bummer.

**Comment by theofidry** [comment]
@sanmai is it actually run _anywhere_?

**Comment by sanmai** [comment]
Not yet, but we can make it, if you insist. Let's get finished with #1106, and I'll help it.

**Comment by maks-rafalko** [comment]
> Bummer. Because I didn't change anything in `src`, it wants to mutate everything.

I've restarted it, but this shouldn't be an issue, because on `push` builds everything is mutated as well.

**Comment by sanmai** [comment]
Fingers crossed. Anyway, merge as you see fit.

**Comment by maks-rafalko** [comment]
Still no luck, same job :|

And now also app veyor is broken (https://github.com/infection/infection/pull/1110#issuecomment-595442625)

**Comment by sanmai** [comment]
There's something seriously wrong with this PR. I'll be having a look later.

File: tests/phpunit/Console/E2ETest.php [hunk 1]
```diff
@@ -140,12 +140,12 @@ public function test_it_runs_on_itself(): void
...
                 'Refusing to run Infection on itself with no memory limit set: it is dangerous.',
                 'To run this test with a memory limit set please use:',
-                'php -dmemory_limit=128M vendor/bin/phpunit --group=large',
+                'php -dmemory_limit=512M vendor/bin/phpunit --group=large',
             ]));
         }
 
         $output = $this->runInfection(self::EXPECT_SUCCESS, [
-            '--test-framework-options="--exclude-group=' . self::EXCLUDED_GROUP . '"',
+            sprintf('--test-framework-options=--exclude-group=%s', self::EXCLUDED_GROUP),
```
Comment (maks-rafalko) [inline tests/phpunit/Console/E2ETest.php]: I'm guessing, but probably quotes affects somehow the build. It's now written in a different way

Comment (sanmai) [inline tests/phpunit/Console/E2ETest.php]:
```suggestion
            '--test-framework-options=--exclude-group=' . self::EXCLUDED_GROUP,
```

Comment (sanmai) [reply tests/phpunit/Console/E2ETest.php]: It didn't help  :see_no_evil:

Comment (maks-rafalko) [reply tests/phpunit/Console/E2ETest.php]: not many cases left to revert :) next is `--no-progress`?

Comment (sanmai) [reply tests/phpunit/Console/E2ETest.php]: It's more line something *isn't added* here.

**Comment by sanmai** [comment]
Curiously, build are broken only for PCOV.

**Comment by sanmai** [comment]
I'm going to close it with hopes I can revisit the idea later.

**Pull request closed without merging**